src/pages/StableSwap/index.tsx

```diff
@@ -245,21 +269,21 @@ export default function StableSwap() {
     (approval === ApprovalState.NOT_APPROVED ||
       approval === ApprovalState.PENDING ||
       (approvalSubmitted && approval === ApprovalState.APPROVED)) &&
-    !(priceImpactSeverity > 3 && !isExpertMode)
+    !(priceImpactSeverity === true && !isExpertMode)
```

Comment:
This is typed as `boolean`, so don't need to check against `true` each time
```suggestion
    !(priceImpactSeverity && !isExpertMode)
```

Maybe for legibility:

```suggestion
    !(hasPriceImpactSeverity && !isExpertMode)
```